django/views/templates/technical_404.html

```diff
@@ -9,9 +9,9 @@
     body * { padding:10px 20px; }
     body * * { padding:0; }
     body { font:small sans-serif; background:#eee; color:#000; }
-    body>div { border-bottom:1px solid #ddd; }
+    body > :where(header, main, footer) { border-bottom:1px solid #ddd; }
```

Comment:
We should do the same in `technical_500.html`.

Reply:
Thanks, done!